RentallCarsAPI/Tools/CarHelper.cs

```diff
@@ -48,5 +49,14 @@ public List<Car> GetAll()
                 return null;
             }
         }
+
+        public Car GetById(Guid id)
+        {
+            var cars = GetAll();
+            if (cars == null)
+                return null;
```

Comment:
Según las convenciones de estilos, se utilizaban siempre los `{` `}` correcto? De todas formas lo podemos tener en cuenta para otro PR
```suggestion
            if (cars == null)
            {
                return null;
            }
```

Reply:
done